ricecooker/classes/nodes.py

```diff
@@ -742,6 +742,8 @@ def validate(self):
             assert self.kind == content_kinds.EXERCISE, "Assumption Failed: Node should be an exercise"
 
             # Check if questions are correct
+            if not any(self.questions):
```

Comment:
Might be better to combine this with the assertion error ("Assumption Failed: Exercise does not have a question ({})".format(self.__dict__))